optuna/trial/_trial.py

```diff
@@ -658,6 +661,11 @@ def _check_distribution(self, name: str, distribution: BaseDistribution) -> None
                 RuntimeWarning,
             )
 
+    def _get_latest_trial(self) -> FrozenTrial:
+        system_attrs = copy.deepcopy(self.storage.get_trial_system_attrs(self._trial_id))
+        self._cached_frozen_trial.system_attrs = system_attrs
+        return self._cached_frozen_trial
```

Comment:
Could you add a todo comment?

```suggestion
    def _get_latest_trial(self) -> FrozenTrial:
        # TODO(eukaryo): Remove this method after `system_attrs` property is deprecated.
        system_attrs = copy.deepcopy(self.storage.get_trial_system_attrs(self._trial_id))
        self._cached_frozen_trial.system_attrs = system_attrs
        return self._cached_frozen_trial
```

Reply:
Agreed.